gen/aircraft.py

```diff
@@ -533,6 +535,17 @@ class AircraftData:
     #: Defines how channels should be named when printed in the kneeboard.
     channel_namer: Type[ChannelNamer] = ChannelNamer
 
+@dataclass
+class AewcInfo:
```

Comment:
Instead of a new class, add the new information to `AwacsInfo`. Something like:

```
@dataclass
class AwacsInfo:
    """AWACS information for the kneeboard."""

    dcsGroupName: str
    callsign: str
    freq: RadioFrequency
    start_time: Optional[timedelta]
    end_time: Optional[timedelta]
```

The callsign is worth including I think. You may need to dig in to pydcs a bit to figure out how that works, but `airsupportgen` might already have the answers.

I think `start_time` and `end_time` are more clear than `departure_time` and `arrival_time`. The way it's currently written those happen to be the same thing, but it would be quite reasonable to change the coverage period to only be while the flight is on station.